tests/test_notify_buyers_to_award_closed_briefs.py

```diff
@@ -135,7 +135,7 @@ def test_send_email_to_brief_user_via_notify_logs_instead_of_sending_for_dry_run
         notify_client.assert_not_called()
 
 
-@mock.patch('dmscripts.notify_buyers_to_award_closed_briefs.DMNotifyClient', autospec=True)
+@mock.patch('dmscripts.notify_buyers_to_award_closed_briefs.scripts_notify_client')
```

Comment:
Is this really the only test where we patch the `DMNotifyClient`? 🤔 I guess all the other ones are top-level scripts, huh.

Reply:
Indeed